### ISSUE TITLE
Mimic Concept2 PM5 display

### ISSUE THREAD

**Issue by cjeanneret** [cjeanneret/gopolloplus#13]
It might be interesting to mimic the PM5 display as shown in the quickstart doc here:
https://www.concept2.com/files/pdf/us/monitors/PM5_QuickStartGuide_EnglishOnly.pdf

- [ ] Understand the available options
- [ ] Create "startup menu"
  - [ ] Chose default display (graphs vs PM5)
  - [ ] Allow to set time or distance goal
  - [ ] "Just row" - must be selected by default, and active once we start rowing
  - [ ] History (show file list, and plot graphs)
- [ ] PM5: grid layout (no graphs afaik, only sad numbers)

**Comment by timcolson** [comment]
<img width="600" alt="image" src="https://user-images.githubusercontent.com/1376728/111419110-17632a00-86a6-11eb-898e-5a58049e5df6.png">

Basic data screen is all text, but there is a [Force Curve](https://www.concept2.com/indoor-rowers/training/tips-and-general-info/using-the-force-curve) graph that looks interesting:


![Force Curve](https://www.concept2.com/files/images/indoor-rowers/training/forcecurve.gif)

<img width="778" alt="image" src="https://user-images.githubusercontent.com/1376728/111419301-77f26700-86a6-11eb-9500-749874628300.png">

**Comment by cjeanneret** [comment]
Hmmm we won't be able to graph the force curve. We just don't have the needed data with the installed sensors. Sad, but... We would need something on the handle or band, with a continuous reading.

**Comment by timcolson** [comment]
Yes, I see what you mean. The console can see the velocity deltas, but FDF USB only sends a point in time readout once per stroke. Bummer.

**Comment by cjeanneret** [comment]
yup. and... well, we can make some physics and maths, but it wouldn't be the reality. I'd rather re-use the "projected" feature - already thought about it, and there might be multiple ways. Mostly used when we set some objectives, like duration or distance to reach.
